H2MLauncher.UI/App.xaml.cs

```diff
@@ -74,6 +86,39 @@ private void ConfigureServices(IServiceCollection services)
             services.AddTransient<MainWindow>();
         }
 
+        private static string GetAppSettingsPath()
+        {
+            string appsettings = Path.Combine(Constants.LocalDir, "appsettings.json");
+            if (IsDevelopment())
+            {
+                appsettings = Path.Combine(Directory.GetCurrentDirectory(), "appsettings.development.json");
+            }
+            return appsettings;
+        }
+
+        private static IConfigurationRoot BuildConfiguration()
+        {
+            string appSettingsFileName = GetAppSettingsPath();
+            if (!File.Exists(appSettingsFileName))
+            {
+                JsonFileHelper.AddOrUpdateSection(appSettingsFileName, nameof(H2MLauncherSettings), new H2MLauncherSettings());
+            }
+            IConfigurationBuilder builder = new ConfigurationBuilder()
+                .AddJsonFile(appSettingsFileName)
```

Comment:
Add embedded appsettings.json resource file

```
public static void AddEmbeddedJsonFile(this IConfigurationBuilder configuration, string resourceName)
{
    var host = Assembly.GetEntryAssembly();
    if (host == null)
        return;

    var fullFileName = $"{host.GetName().Name}.{resourceName}";
    using var input = host.GetManifestResourceStream(fullFileName);
    if (input != null)
    {
        configuration.AddJsonStream(input);
    }
}
```

And add the development one on top, then the one from localappdata